src/webpack/loader.js

```diff
@@ -30,10 +30,10 @@ function writeFileIfChanged(outputFilename, cssText) {
 }
 
 const startMarker = 'ZACS_MAGIC_CSS_STYLESHEET_MARKER_START("'
+const cssDisclaimer = '/* Generated CSS file (from ZACS Stylesheets) - do not edit! */\n'
 
 exports.default = function loader(source, inputSourceMap) {
-  // TODO: Options
-  // const options = getOptions(this)
+  const { cacheDirectory = '.zacs-cache', extension = '.zacs.css' } = loaderUtils.getOptions(this) || {}
```

Comment:
Maybe it woud be a good idea to hide the `.zacs-cache` folder inside `node_modules/@nozbe/zacs`, so we don't pollute the project's root?

Reply:
Absolutely not, this is a bad practice. node_modules should be treated as immutable (it breaks many things not to do this), and cache folders should be discoverable by users so they can be inspected, and removed manually/by script if desired.